Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM python:3.6.8-alpine3.8 as base
+FROM python:3.6.10-alpine as base
 FROM base as builder
 RUN apk --no-cache --update-cache add gcc python python-dev py-pip build-base wget
 RUN ln -s /usr/include/locale.h /usr/include/xlocale.h
 RUN pip install pipenv
 RUN mkdir -p /src/ngsi-timeseries-api
 COPY Pipfile /src/ngsi-timeseries-api/Pipfile
 COPY Pipfile.lock /src/ngsi-timeseries-api/Pipfile.lock
-RUN mkdir /install
-WORKDIR /install
 RUN cd /src/ngsi-timeseries-api && { pipenv lock -r > /requirements.txt; }
-RUN pip install --install-option="--prefix=/install" -r /requirements.txt
+RUN pip install -r /requirements.txt
```

Comment:
not sure why, but installation in the old way was not working anymore, now it's working, but as result the image is bigger :/

Reply:
see comment here

Reply:
@c0c0n3

Reply:
thx!